### PR TITLE
Interative To-Do-List Testing

### PR DESCRIPTION
**Pull request by AyomipoMajek**
- [ ] Testing Edit task
- [ ] Testing Edit task on Local Storage
- [ ] Testing Completed task
- [ ] Testing Clear completed

### CODE REVIEW

File: src/validate.test.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    const todo = clearCompleted(toDos);
+    window.localStorage.setItem('todo', JSON.stringify(todo));
+    const result = JSON.parse(window.localStorage.getItem('todo'));
+    expect(result.length).toBe(0);
+  });
+  test('Clear all completed tasks from the DOM', () => {
+    const toDos = JSON.parse(window.localStorage.getItem('todo'));
+    const todo = clearCompleted(toDos);
+    window.localStorage.setItem('todo', JSON.stringify(todo));
+    // eslint-disable-next-line no-unused-vars
```
Comment (Gambit142) [inline src/validate.test.js:132]: - [ ] Kindly remove the Eslint disabling comment. Eslint helps us to write clean, maintainable and scalable codes. Using an Eslint disabling comment defeats this purpose. Kindly remove it. 👍

File: src/validate.test.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+  test('Edit todo object in local storage', () => {
+    const editTodoOnSpy = jest.spyOn(Display, 'editTodo');
+    Display.editTodo(0);
+    expect(editTodoOnSpy).toHaveBeenCalledTimes(1);
+    const result = JSON.parse(window.localStorage.getItem('todo'));
+    expect(result.length).toBe(2);
+    expect(result[0].text).toBe('Visit the gym with Andre');
+  });
+});
```
Comment (Gambit142) [inline src/validate.test.js:44]: - [ ] The test for editing tasks was not properly written. You are testing for the task that was inputted and not a task that was edited. To successfully test the function for editing a task, you need to change the description for a task and check if it returns the edited task. Kindly implement this change 👍 💻

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @AyomipoMajek 👋, 

Good job so far!

## Highlights
- All checks passed ✔️ 
- Tested the function for editing ✔️ 
- Tested the function for changing the status of a task ✔️ 
- Tested the function for clearing all completed tasks ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
#### I can't highlight the relevant code in the _**README.md**_ file that is the reason why I am leaving this suggestion here. 

- [ ] The link to your **_MIT LICENSE_** is broken. It displays an error page when it is clicked. Kindly fix the link. 👍 

_Kindly check the comments under the review for other required changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by jerryowusu — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
All tests are passing :heavy_check_mark: 
Great effort so far :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**